lib/ansible/config/base.yml

```diff
@@ -1897,6 +1897,24 @@ SHOW_CUSTOM_STATS:
   ini:
   - {key: show_custom_stats, section: defaults}
   type: bool
+SSH_AGENT:
```

Comment:
what about using a bit more generic names for base config, 'agent' .. in case we want to add gpg-agent/keychain or others for other contexts?

we can always have an override to specific agent in the plugins, though they would not benefit from the 'early' startup